packages/graph/hash_graph/lib/graph/src/store/postgres/mod.rs

```diff
@@ -653,80 +685,219 @@ impl PostgresStore<Transaction<'_>> {
 
     #[doc(hidden)]
     #[cfg(feature = "__internal_bench")]
-    async fn insert_entity_batch_by_type(
+    async fn insert_entity_editions(
         &self,
-        entity_uuids: impl IntoIterator<Item = EntityUuid, IntoIter: Send> + Send,
-        entities: impl IntoIterator<Item = EntityProperties, IntoIter: Send> + Send,
-        link_datas: impl IntoIterator<Item = Option<LinkData>, IntoIter: Send> + Send,
+        entities: impl IntoIterator<
+            Item = (EntityProperties, Option<LinkOrder>, Option<LinkOrder>),
+            IntoIter: Send,
+        > + Send,
         entity_type_version_id: VersionId,
-        owned_by_id: OwnedById,
-        updated_by_id: UpdatedById,
-    ) -> Result<u64, InsertionError> {
+        actor_id: UpdatedById,
+    ) -> Result<Vec<i64>, InsertionError> {
+        self.client
+            .simple_query(
+                "CREATE TEMPORARY TABLE entity_editions_temp (
```

Comment:
If you don't want to explicitly drop this at the end, you can use `ON COMMIT DROP`

Reply:
Oh, that's nice, thanks!

Reply:
Sadly, this didn't work, the table was not dropped, probably because we are in a big transaction. I left the `DROP` statement in.